src/js/planting.js

```diff
@@ -6,7 +6,7 @@ import SessionDataModel from 'session-data';
 import ManifestoDataModel from 'manifesto-data';
 import Main from 'module/main/main';
 import Plant from 'module/plant/plant';
-import Toolbox from 'module/toolbox/toolbox';
+import Sidebar from './module/toolbox/sidebar';
```

Comment:
Is there a reason for `.` on the beggining of path? If it exists, we need to establis some convension about it.